content/en/docs/tasks/manage-hugepages/scheduling-hugepages.md

```diff
@@ -6,10 +6,10 @@ content_template: templates/task
 ---
 
 {{% capture overview %}}
-{{< feature-state state="beta" >}}
+{{< feature-state state="GA" >}}
```

Comment:
```suggestion
{{< feature-state state="stable" >}}
```
The [Hugo shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state) for this uses the keyword `stable`